src/flex/FlexView.js

```diff
@@ -43,6 +43,7 @@ export default React.createClass({
       React.PropTypes.number,
       React.PropTypes.string
     ]),
+    events: React.PropTypes.object,
```

Comment:
All of the supported html attributes: https://facebook.github.io/react/docs/tags-and-attributes.html#html-attributes
But there shouldn't be any "special" props other than maybe input props: https://facebook.github.io/react/docs/forms.html#interactive-props

Looking at other props, `id` is receiving exactly the same treatment as `events` props

Naming still confuses me, but maybe it's just me :P @gabro thoughts?